src/js/edu-benefits/routes.jsx

```diff
@@ -87,6 +87,23 @@ export default function createRoutes(store) {
       },
     },
     {
+      path: '1990-rjsf',
+      indexRoute: { onEnter: (nextState, replace) => replace('/1990-rjsf/introduction') },
+      component: asyncLoader(() => {
```

Comment:
Eh, I guess I should do that. It doesn't matter as much here because we already have approval for the 1990 and there's no way for a user to get to this version of the form.